internal/config/cgmgr/systemd.go

```diff
@@ -46,6 +46,24 @@ func (*SystemdManager) ContainerCgroupPath(sbParent, containerID string) string
 	return parent + ":" + crioPrefix + ":" + containerID
 }
 
+// ContainerCgroupAbsolutePath takes arguments sandbox parent cgroup and container ID and
+// returns the cgroup path on disk for that containerID. If parentCgroup is empty, it
+// uses the default parent system.slice
+func (r *SystemdManager) ContainerCgroupAbsolutePath(sbParent, containerID string) (string, error) {
+	parent := defaultSystemdParent
+	if sbParent != "" {
+		parent = sbParent
+	}
+	logrus.Debugf("Expanding systemd cgroup slice %v", parent)
+	cgroup, err := systemd.ExpandSlice(parent)
+	if err != nil {
+		return "", errors.Wrapf(err, "error expanding systemd slice to get container %s stats", containerID)
+	}
+
+	cgroup = filepath.Join(cgroup, crioPrefix+"-"+containerID) + ".scope"
```

Comment:
nit, we're already adding the strings together inside of the call, I think we should do the same with scope
`crioPrefix+"-"+containerID+".scope"`

Reply:
fixed!